README.md

```diff
@@ -56,61 +62,69 @@ There is a german article about [Security DevOps – Angreifern (immer) einen Sc
 
 ### Prerequisites
 
-- kubernetes (last 4 mayor releases supported: `1.15`, `1.16`, `1.17` & `1.18`)
+- kubernetes (last 4 major releases supported: `1.15`, `1.16`, `1.17` & `1.18`)
 
 ### Deployment (based on Helm)
 
+Deploy the secureCodeBox operator first:
+
 ```bash
-# Deploy the secureCodeBox Operator first
```

Comment:
Shouldn't we separate the bash block also for the "# Deploy SCB scanner Charts" headline too? Otherwise it's a little bit inconsistent 🤔

Reply:
Done